hashuri/hashuri.cpp

```diff
@@ -0,0 +1,70 @@
+//https://www.infoarena.ro/problema/hashuri
+
+#include <iostream>
+#include <vector>
+#include <fstream>
+
+using namespace std;
+
+vector <int> hashTable[1000000];
+int MOD = 999961;
+
+void addParameter(int parameter) {
+    int key, count = 0;
+    key = parameter % MOD;
+    for (int i = 0; i < hashTable[key].size(); ++i) {
+        if (parameter == hashTable[key][i]) {
+            break;
+        } else {
+            count += 1;
+        }
+    }
+    if (count == hashTable[key].size()) {
+        hashTable[key].push_back(parameter);
+    }
```

Comment:
No need for this whole thing, you have a find method that you can use. (the type3 you implemented)